cmake/modules/SearchRootCoreDeps.cmake

```diff
@@ -37,9 +37,10 @@ if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.14)
     set(PYTHON_EXECUTABLE "${Python3_EXECUTABLE}" CACHE INTERNAL "" FORCE)
     set(PYTHON_INCLUDE_DIRS "${Python3_INCLUDE_DIRS}" CACHE INTERNAL "" FORCE)
     set(PYTHON_LIBRARIES "${Python3_LIBRARIES}" CACHE INTERNAL "" FORCE)
+    set(PYTHON_VERSION_STRING "${Python3_VERSION}" CACHE INTERNAL "" FORCE)
     set(PYTHON_VERSION_MAJOR "${Python3_VERSION_MAJOR}" CACHE INTERNAL "" FORCE)
     set(PYTHON_VERSION_MINOR "${Python3_VERSION_MINOR}" CACHE INTERNAL "" FORCE)
-    set(PYTHON_VERSION_STRING "${Python3_VERSION_MAJOR}_${Python3_VERSION_MINOR}" CACHE INTERNAL "" FORCE)
+    set(DUNDER_PYTHON_VERSION_STRING "${Python3_VERSION_MAJOR}_${Python3_VERSION_MINOR}" CACHE INTERNAL "" FORCE)
```

Comment:
Dunder is an abbreviation of "double underscore", but here there is only one, so I would pick another name. `PYTHON_UNDERSCORE_VERSION_STRING` ? A bit long but... can't think of anything better.

Reply:
`PYTHON_UNDER_VERSION_STRING` ? @oshadura what do you think?

Reply:
PYTHON_INSTALL_VERSION_STRING ?

Reply:
we need definitelly to add description what does this variable do @maxgalli